Homestead/readme.md

```diff
@@ -159,7 +159,11 @@ Add the following function to your `.bashrc` or `.bash_profile`:
 
 ```bash
 function homestead() {
-  ( cd ~/Code/homestead && vagrant $* )
+  DIRECTORY=$(pwd)
+  HOMESTEAD_DIRECTORY="$HOME/Code/homestead"
```

Comment:
One thing to note, maybe in the directions is that this depends on having the directory containing the homestead repo be called `Code/`? 🤷‍♂️

Reply:
It does, right below!

> Make sure the path to the homestead repository (`~/Code/homestead`) matches the path you chose when installing Homestead installation on your local machine.

Reply:
Ha! Totes missed it 🤦‍♂️